SR830/SR830.py

```diff
@@ -386,7 +386,20 @@ def wait_time(self, c):
             returnValue(9*tc)
         else:# slope == 3:
             returnValue(10*tc)
-    
+
+    @setting(35, 'Output Overload', returns='b')
+    def output_overload(self, c):
+        ''' Gets the output overload status bit
```

Comment:
nit: For consistency with other docstrings in this file, use triple double quotes. Also, no space after quotes on the first line and end it with a period.